rules/terraformrules/terraform_empty_list_equality_test.go

```diff
@@ -30,6 +30,25 @@ resource "aws_db_instance" "mysql" {
 						End:      hcl.Pos{Line: 3, Column: 18},
 					},
 				},
+			},
+		},
+		{
+			Name: "comparing with [] multiple times in the same statement is not recommended",
```

Comment:
You can tighten up these names to be shorter/clearer for Go test output. For example, `"multiple empty comparisons"`.